nginxConfig/nginx.conf

```diff
@@ -10,5 +10,6 @@ server {
                 proxy_set_header            onboarding-ip ${DOLLAR}proxy_add_x_forwarded_for;
                 proxy_set_header            x-api-key ${ONBOARDING_X_API_KEY};
                 proxy_ssl_server_name       on;
+                resolver                    169.254.169.253 valid=60s;
```

Comment:
wondering if this should be an ONBOARDING_xxxx env variable similar to how we setup proxy_pass and proxy_set_header

Reply:
I'm ok with that since this is specific to AWS. Could you add a quick comment explaining that's the Route53 DNS resolver?

Reply:
this comment explaining the DNS resolver would fit well in the README "Nginx server" chapter.
![image](https://github.com/1NCE-GmbH/1nce-iot-device-onboarding/assets/99891188/67fe8314-4c10-40be-8651-c888f936de91)